apps/editors/tests/test_views.py

```diff
@@ -2330,6 +2330,20 @@ def test_viewing_queue(self):
         data = json.loads(r.content)
         eq_(data[str(self.addon.id)], self.editor.display_name)
 
+    def test_platform_text(self):
+        version = Version.objects.create(addon=self.addon, version='0.2')
+        version.created = datetime.today() + timedelta(days=1)
```

Comment:
Can you try without this line and the following, just to make sure the test still succeeds?